server/services/grid_service.py

```diff
@@ -0,0 +1,34 @@
+import geojson
+from shapely.geometry import shape
+from shapely.prepared import prep
+
+
+class GridService:
+
+    @staticmethod
+    def geometries_intersect(tile: shape, aoi: shape) -> bool:
+        """
+        Takes two shapes and returns True if the intersect, False if not
+        :param tile: shapely.geometry.shape
+        :param aoi: shapely.geometry.shape
+        :return: bool
+        """
+        p = prep(aoi)
```

Comment:
Single letter variable 2 demerit points 👎 😉 
Rename to something more descriptive